package.json

```diff
@@ -31,13 +31,13 @@
     "bootstrap-sass": "^3.3.5",
     "classnames": "^2.1.3",
     "event-source-polyfill": "^0.0.7",
-    "highlight.js": "^8.9.1",
+    "highlight.js": "^9.3.0",
     "lodash": "^4.11.2",
     "moment": "^2.10.6",
     "postal": "^2.0.0",
     "postal.diagnostics": "^0.7.4",
-    "react": "^0.13.3",
-    "react-highlight": "^0.5.1",
+    "react": "^15.0.2",
```

Comment:
FYI React is why I left https://github.com/Glimpse/Glimpse.Client.Prototype/issues/74 open, even though everything else has been updated.